frontend/app/src/components/App.svelte

```diff
@@ -73,104 +73,113 @@
+                console.log("Failed to add hot group exclusion", chatId);
+            }
+        });
     }
 
     function deleteFrozenGroup(chatId: string): void {
-        client
-            .deleteFrozenGroup(chatId)
-            .then((success) => {
-                if (success) {
-                    console.log("Group deleted", chatId);
-                } else {
-                    console.log("Failed to delete frozen group", chatId);
-                }
-            });
+        client.deleteFrozenGroup(chatId).then((success) => {
+            if (success) {
+                console.log("Group deleted", chatId);
+            } else {
+                console.log("Failed to delete frozen group", chatId);
+            }
+        });
     }
 
     function freezeGroup(chatId: string, reason: string | undefined): void {
-        client
-            .freezeGroup(chatId, reason)
-            .then((success) => {
-                if (success) {
-                    console.log("Group frozen", chatId);
-                } else {
-                    console.log("Failed to freeze group", chatId);
-                }
-            });
+        client.freezeGroup(chatId, reason).then((success) => {
+            if (success) {
+                console.log("Group frozen", chatId);
+            } else {
+                console.log("Failed to freeze group", chatId);
+            }
+        });
     }
 
     function removeHotGroupExclusion(chatId: string): void {
-        client
-            .removeHotGroupExclusion(chatId)
-            .then((success) => {
-                if (success) {
-                    console.log("Hot group exclusion removed", chatId);
-                } else {
-                    console.log("Failed to remove hot group exclusion", chatId);
-                }
-            });
+        client.removeHotGroupExclusion(chatId).then((success) => {
+            if (success) {
+                console.log("Hot group exclusion removed", chatId);
+            } else {
+                console.log("Failed to remove hot group exclusion", chatId);
+            }
+        });
     }
 
     function unfreezeGroup(chatId: string): void {
-        client
-            .unfreezeGroup(chatId)
-            .then((success) => {
-                if (success) {
-                    console.log("Group unfrozen", chatId);
-                } else {
-                    console.log("Failed to unfreeze group", chatId);
-                }
-            });
+        client.unfreezeGroup(chatId).then((success) => {
+            if (success) {
+                console.log("Group unfrozen", chatId);
+            } else {
+                console.log("Failed to unfreeze group", chatId);
+            }
+        });
+    }
+
+    function deleteMessage(
+        chatId: string,
+        messageId: bigint,
+        threadRootMessageIndex?: number | undefined
+    ): void {
+        client.deleteMessage(chatId, threadRootMessageIndex, messageId).then((success) => {
```

Comment:
I think we'll need to pass up an `as_super_admin` flag, otherwise the BE won't know that it should check if the user is a super admin.